### PR TITLE
New SecurityBaseline module PoC with 19 security baseline checks in

### PR DESCRIPTION
**Pull request by MariusNi**
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] I added unit-tests to validate my changes. All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

**Review by robertschaedler3 — DISMISSED** [review]
👍🏻

File: src/modules/mim/securitybaseline.json [hunk 1]
```diff
@@ -7,51 +7,147 @@
       "type": "mimComponent",
       "contents": [
         {
-          "name": "RemediateSecurityBaseline",
+          "name": "AuditSecurityBaseline",
```
Comment (simonjaeger) [inline src/modules/mim/securitybaseline.json:10]: >AuditSecurityBaseline [](http://example.com/codeflow?start=19&length=21)

nit: Many MIM objects in this file are not camel case. Same for the DTMI file.

**Review by simonjaeger — APPROVED** [review]
:shipit:

**Review by robertschaedler3 — APPROVED** [review]
👍🏻

**Review by simonjaeger — APPROVED** [review]
:shipit:

**Pull request merged**